thonny/plugins/coloring.py

```diff
@@ -112,6 +112,7 @@ def _raise_tags(self):
         self.text.tag_raise("open_string3")
         self.text.tag_raise("open_string")
         self.text.tag_raise("sel")
+        self.text.tag_raise("builtin")
```

Comment:
I think it's better to raise "builtin" over function calls (ie. the two argument variant of `tag_raise`. Tag `sel` (for selected) text should be the topmost, othewise the selection could get illegible foreground.

Reply:
Ok.